### ISSUE TITLE
implement mAP

### ISSUE THREAD

**Issue by gabriben** [gabriben/multilabel#82]
- [ ] in tensorflow
- [ ] sigmoid
- [ ] ema?

**Comment by gabriben** [comment]
pytorch

- [ ] rewrite sigmoidF1 in pytorch
- [ ] make PASCAL-VOC work

**Comment by gabriben** [comment]
- [ ] all the elements here: https://github.com/Alibaba-MIIL/ASL/issues/30
